canopen/pdo/__init__.py

```diff
@@ -0,0 +1,62 @@
+from .base import PdoBase, Maps, Map, Variable
+
+import logging
+import collections
+import itertools
+import canopen
+
+logger = logging.getLogger(__name__)
+
+
+class PDO(PdoBase):
```

Comment:
Could you also add the `read()` and `save()` methods?

Reply:
The ``read()`` and ``save()`` methods are available from the PdoBase.